test/regress/cli/regress1/datatypes/nullables/nullables2.smt2

```diff
@@ -0,0 +1,8 @@
+(set-logic ALL)
+(set-info :status unsat)
```

Comment:
All `unsat` benchmarks with `Nullable` will fail until we add them to the proof signatures for ALF/LFSC.

For now, add `; DISABLE-TESTER: proof` to this benchmark (and any other unsat ones with Nullable).

Reply:
Done